views/ViewChart.react.js

```diff
@@ -28,24 +24,15 @@ const ViewChart = ({ graph, classes }) => {
           <Typography variant="subtitle1">
             <pre style={{ fontFamily: 'inherit' }}>{description}</pre>
           </Typography>
-          <Button
-            variant="outline"
-            color="default"
-            className={classes.showFilterButton}
-            onClick={() => openFilterMenu()}
-          >
-            <FilterListIcon className={classes.leftIcon} />
-            Filters
-          </Button>
         </div>
       )}
-      <Drawer anchor="left" open={showFilterMenu} onClose={onCloseFilterMenu}>
+      <div style={{ paddingBottom: '25px' }}>
```

Comment:
Should this be in the CSS file?

Reply:
Thank you for this. 
https://github.com/AMP-SCZ/dpdash/pull/252/commits/e62b6cca672da42cca9aab1f550d26e78d59e51c